### ISSUE TITLE
dot chart doesnt work fine with big labels

### ISSUE THREAD

**Issue by brodock** [DmitryBaranovskiy/g.raphael#165]
If I create a dotchart with a big label, it shrinks the whole graph, not only the corresponding dimension.
For example, if I use a big label on Y axis, it should only shrink the width, but maintain the height untouched.

I had been able to track what I think is the root cause of the problem, and may be related to the gutter recalculation after axis labels generation.

We should store gutter as a two dimension data, and use the correct dimension to calculate circles position.

**Comment by jeffchuber** [comment]
I'm also curious what happens when the x axis labels get too long. It appears that it actually just destroys the next label preventing it from being written at all if the spacing is too smal to accomodate it. I would rather the x axis labels be able to be rotated 45 degrees so that it could support much longer strings. I haven't found the root of the error - still experimenting. any thoughts @brodock ?

**Comment by brodock** [comment]
Yes, we are playing with the code and figuring out the best solution for it. We will share it as soon as we get an acceptable one.

**Comment by jeffchuber** [comment]
Thanks!
